lib/throttle_helpers/src/throttle_helpers.cpp

```diff
@@ -42,5 +42,5 @@ bool DoPotentiometersAgree(uint16_t left_acc_pos, uint16_t right_acc_pos)
  */
 bool BrakeAndAccelerator(uint16_t brake_position, uint16_t throttle_percent)
 {
-    return brake_position > 0 && throttle_percent > 0;
+    return brake_position > 5 && throttle_percent > 0;
```

Comment:
Looks good but might want to set a variable thats equal to 5 instead of brake_position > 5 (just to make things more readable).